tests/components/Microsoft.DotNet.UpgradeAssistant.Tests/UpgraderTests.cs

```diff
@@ -51,7 +51,7 @@ public async Task UpgraderStepsEnumeration()
             // in the expected order
             using var context = mock.Mock<IUpgradeContext>().Object;
 
-            var steps = upgrader.GetStepsForContext(context);
+            var steps = upgrader.AllSteps;
```

Comment:
We don't need conditional steps in tests?

Reply:
We could leave this; I pulled it out because the test doesn't have an steps that don't apply so it was just an extra call that didn't do anything.